modules/xmpp/ChatRoom.js

```diff
@@ -747,26 +778,35 @@ export default class ChatRoom extends Listenable {
         }
     }
 
+    /**
+     * Extracts the features from the presence.
+     * @param node the node to process.
+     * @param features the Set of features where extracted data will be added.
+     * @private
+     */
+    _extractFeatures(node, features) {
+        for (let j = 0; j < node.children.length; j++) {
+            const { attributes } = node.children[j];
+
+            if (attributes && attributes.var) {
+                features.add(attributes.var);
+            }
+        }
+    }
+
     /**
      * Initialize some properties when the focus participant is verified.
      * @param from jid of the focus
-     * @param mucJid the jid of the focus in the muc
+     * @param features the features reported in jicofo presence
      */
-    _initFocus(from, mucJid) {
+    _initFocus(from, features) {
         // skip if we have queried jicofo already, it will not change
```

Comment:
Now that we don't use disco#info can we remove this check?

Reply:
Ah, yeah. It was a merge conflict I got and that had sneaked in.